services/slam/client_test.go

```diff
@@ -299,6 +332,11 @@ func TestFailingClient(t *testing.T) {
 		test.That(t, err.Error(), test.ShouldContainSubstring, "failure to get latest map info")
 		test.That(t, timestamp, test.ShouldResemble, time.Time{})
 
+		// test properties
+		prop, err := failingSLAMClient.Properties(context.Background())
+		test.That(t, err.Error(), test.ShouldContainSubstring, "failure to get properties")
```

Comment:
could you make this error a variable and test that the errors are the same with `test.ShouldBeError`

Reply:
Attempted to switch to`test.ShouldBeError` when using `errors.Wrapf` but received error:

```
    client_test.go:346: Expected: 'failure to get properties: failure to get properties'
        Actual:   'failure to get properties: rpc error: code = Unknown desc = failure to get properties'
        (Should be equal)
        Diff:     'failure to get properties: rpc error: code = Unknown desc = failure to get properties'
```
The `rpc error: code = Unknown desc =` is produced due to the pathway from client to slam function. I tried added it in multiple places without success so am leaving it as the simple, `test.ShouldContainSubstring` which matches the pattern of the rest of the file.